### PR TITLE
Fix: Dispose Graphics and Image in SvgRenderer.

### PR DESCRIPTION
**Pull request by H1Gdev**
Dispose Graphics and/or Image in SvgRenderer if needed.

### CODE REVIEW

File: Source/SvgDocument.cs [hunk 1]
```diff
@@ -545,21 +539,35 @@ public virtual Bitmap Draw(int rasterWidth, int rasterHeight)
...
+            this.RasterizeDimensions(ref bitmapSize, rasterWidth, rasterHeight);
 
             if (bitmapSize.Width == 0 || bitmapSize.Height == 0)
                 return null;
 
-            var bitmap = new Bitmap((int)Math.Round(bitmapSize.Width), (int)Math.Round(bitmapSize.Height));
+            Bitmap bitmap = null;
             try
             {
-                var renderer = SvgRenderer.FromImage(bitmap);
```
Comment (H1Gdev) [inline Source/SvgDocument.cs]: Do not Dispose at latest version.

File: Source/Rendering/SvgRenderer.cs [hunk 2]
```diff
@@ -139,13 +152,14 @@ public static ISvgRenderer FromImage(Image image)
         /// <param name="graphics">The <see cref="Graphics"/> to create the renderer from.</param>
         public static ISvgRenderer FromGraphics(Graphics graphics)
         {
-            return new SvgRenderer(graphics);
+            return new SvgRenderer(graphics, false);
```
Comment (H1Gdev) [inline Source/Rendering/SvgRenderer.cs]: Do not Dispose.

File: Source/SvgDocument.cs [hunk 3]
```diff
@@ -461,46 +466,43 @@ public void Draw(Graphics graphics, SizeF? size)
                 throw new ArgumentNullException("graphics");
```
Comment (H1Gdev) [inline Source/SvgDocument.cs]: What is this method used for ?
Not `SizeF` but `RectangleF` ?

Comment (mrbean-bremen) [reply Source/SvgDocument.cs]: see #337

**Comment by mrbean-bremen** [comment]
@H1Gdev - is this ready for merge? Can you describe the problem you solved?

**Comment by H1Gdev** [comment]
@mrbean-bremen 

I analyzed source codes with focus on SvgRenderer at after #433.

I fixed it because it is an instances of IDisposable but not call Dispose method.
I checked that called Dispose when call SvgDocument.Draw().

File: Source/SvgDocument.cs (hunk 3 above)
Comment (H1Gdev) [reply Source/SvgDocument.cs]: @mrbean-bremen 

Thanks!!

I test following source code.
```cs
var imageSize = svgDoc.GetDimensions();

// same size
using (var image = new Bitmap((int)imageSize.Width, (int)imageSize.Height))
using (var graphics = Graphics.FromImage(image))
{
    var size = new SizeF(imageSize.Width, imageSize.Height);
    svgDoc.Draw(graphics, size);
}

// size up
using (var image = new Bitmap((int)imageSize.Width, (int)imageSize.Height))
using (var graphics = Graphics.FromImage(image))
{
    var size = new SizeF(imageSize.Width * 2, imageSize.Height * 2); 
    svgDoc.Draw(graphics, size);
}

// size down
using (var image = new Bitmap((int)imageSize.Width, (int)imageSize.Height))
using (var graphics = Graphics.FromImage(image))
{
    var size = new SizeF(imageSize.Width / 2, imageSize.Height / 2);
    svgDoc.Draw(graphics, size);
}
```
The results were all the same.

Comment (mrbean-bremen) [reply Source/SvgDocument.cs]: ok, that doesn't seem right...

**Pull request merged**